docs/management/managing-saved-objects.asciidoc

```diff
@@ -5,7 +5,7 @@ The *Saved Objects* UI helps you keep track of and manage your saved objects. Th
 store data for later use, including dashboards, visualizations, maps, index patterns,
 Canvas workpads, and more.
 
-To get started, open the menu, then go to *Stack Management > {kib} > Saved Objects*. With this UI, you can:
+To get started, open the menu, then click *Stack Management > Saved Objects*. With this UI, you can:
```

Comment:
Remove UI? That might be a separate PR because I see it in other places as well.

Reply:
It's everywhere! But I'll remove it here.